hieroplatform/src/test/java/org/hiero/sketch/QuantileSketchTest.java

```diff
@@ -1,36 +1,54 @@
 package org.hiero.sketch;
 
-import org.hiero.sketch.spreadsheet.OrderedColumn;
+import org.hiero.sketch.spreadsheet.ColumnOrientation;
 import org.hiero.sketch.spreadsheet.QuantileList;
 import org.hiero.sketch.spreadsheet.QuantileSketch;
+import org.hiero.sketch.table.ColumnSortOrder;
+import org.hiero.sketch.table.ListComparator;
 import org.hiero.sketch.table.Table;
 import org.junit.Test;
 
-import java.util.ArrayList;
-import java.util.List;
-
+import static junit.framework.TestCase.assertTrue;
 import static org.hiero.sketch.TableTest.getIntTable;
 
 public class QuantileSketchTest {
 
     @Test
     public void testQuantile() {
         final int numCols = 2;
-        final List<OrderedColumn> sortOrder = new ArrayList<OrderedColumn>(numCols);
-        for (int i = 0; i < numCols; i++) {
-            sortOrder.add(i, new OrderedColumn("Column" + String.valueOf(i), ((i % 2) == 0)));
-        }
         final int leftSize = 1000;
         final Table leftTable = getIntTable(leftSize, numCols);
-        final int rightSize = 1000;
+        ColumnSortOrder cso = new ColumnSortOrder();
+        for (String colName: leftTable.schema.getColumnNames()) {
+            cso.append(new ColumnOrientation(leftTable.schema.getDescription(colName), true));
+        }
+
+        final int rightSize = 2000;
         final Table rightTable = getIntTable(rightSize, numCols);
-        final QuantileSketch qSketch = new QuantileSketch(sortOrder);
-        final int resolution = 10;
-        final QuantileList leftQ = qSketch.getQuantile(leftTable, resolution);
-        //System.out.println(leftQ.toString());
-        final QuantileList rightQ = qSketch.getQuantile(rightTable, resolution);
-        //System.out.println(rightQ.toString());
-        final QuantileList mergedQ = qSketch.mergeQuantiles(leftQ, rightQ);
-        //System.out.println(mergedQ.toString());
+        final int resolution = 100;
+        final QuantileSketch qSketch = new QuantileSketch(cso, resolution);
+        final QuantileList leftQ = qSketch.getQuantile(leftTable);
+        final ListComparator leftComp = cso.getComparator(leftQ.quantile);
+        //System.out.println(leftQ);
+        for(int i = 0; i < leftQ.getQuantileSize() -1; i++ )
+            assertTrue(leftComp.compare(i,i+1) <= 0);
+        final QuantileList rightQ = qSketch.getQuantile(rightTable);
+        //System.out.println(rightQ);
```

Comment:
Just a nit. Please remove all commented out System.out.printlns() from all the affected files.